lib/plugins/aws/package/lib/generateCoreTemplate.js

```diff
@@ -54,27 +54,30 @@ module.exports = {
         });
     }
 
+    const mergeProperties = [];
+
     if (isS3TransferAccelerationEnabled && isS3TransferAccelerationSupported) {
       // enable acceleration via CloudFormation
-      this.serverless.service.provider.compiledCloudFormationTemplate
-        .Resources.ServerlessDeploymentBucket.Properties = {
-          AccelerateConfiguration: {
-            AccelerationStatus: 'Enabled',
-          },
-        };
+      mergeProperties.push({
+        AccelerateConfiguration: {
+          AccelerationStatus: 'Enabled',
+        },
+      });
       // keep track of acceleration status via CloudFormation Output
       this.serverless.service.provider.compiledCloudFormationTemplate
-      .Outputs.ServerlessDeploymentBucketAccelerated = { Value: true };
+        .Outputs.ServerlessDeploymentBucketAccelerated = { Value: true };
     } else if (isS3TransferAccelerationDisabled && isS3TransferAccelerationSupported) {
       // explicitly disable acceleration via CloudFormation
-      this.serverless.service.provider.compiledCloudFormationTemplate
-        .Resources.ServerlessDeploymentBucket.Properties = {
-          AccelerateConfiguration: {
-            AccelerationStatus: 'Suspended',
-          },
-        };
+      mergeProperties.push({
+        AccelerateConfiguration: {
+          AccelerationStatus: 'Suspended',
+        },
+      });
     }
 
+    Object.assign(this.serverless.service.provider.compiledCloudFormationTemplate
+      .Resources.ServerlessDeploymentBucket.Properties, ...mergeProperties);
```

Comment:
`...` is not supported in Node.js 4 (which we still have to support 😞)

Reply:
Thanks, that makes sense, I'll sort that.